### PR TITLE
Feature/hardhat unittest

### PR DESCRIPTION
**Pull request by zorancuc**
hardhat configuration
unit-test of tokens

### CODE REVIEW

File: contracts/FxChild.sol [hunk 1]
```diff
@@ -1,5 +1,6 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
+import "hardhat/console.sol";
```
Comment (imyourm8) [inline contracts/FxChild.sol]: please remove logging from all contracts, simulation contracts can keep logs as they are test related

File: test/shared/utilities.ts [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  //     { jsonrpc: '2.0', method: 'evm_mine', params: [timestamp] },
+  //     (error: any, result: any): void => {
+  //       if (error) {
+  //         reject(error);
+  //       } else {
+  //         resolve(result);
+  //       }
+  //     },
+  //   );
+  // });
```
Comment (imyourm8) [inline test/shared/utilities.ts]: remove comments

**Review by imyourm8 — CHANGES_REQUESTED** [review]
Looks good, just need some clean up from logging in contracts and comments in the code - we don't need commented code when we have git for the history

File: test/shared/utilities.ts (hunk 2 above)
Comment (zorancuc) [reply test/shared/utilities.ts]: OK

**Comment by jdkanani** [comment]
Can you fix the conflict and make sure ci works

**Comment by jdkanani** [comment]
Adding @wschwab

**Pull request closed without merging**